assets/component-facets.css

```diff
@@ -233,6 +233,10 @@
   right: 0;
 }
 
+.facets-vertical .facets__disclosure[open] .facets__summary .icon-caret {
```

Comment:
Only because we are only rotating for the vertical filter 🤔 Unless youd like to do it for the horizontal too cc @duygukalaycioglu

Reply:
Good question! The decision on rotating the arrow is a decision made during mega menu project. 

I would park the horizontal filters as is. There are other arrows who don't rotate right now. It would make sense to create a separate ticket to rotate all arrows. cc: @nicklepine

Reply:
I am aligned 👍